spec/models/miq_user_role_spec.rb

```diff
@@ -207,18 +207,25 @@
     end
   end
 
-  it "deletes with no group assigned" do
-    role = FactoryBot.create(:miq_user_role, :name => "test role")
-    role.destroy
-    expect(MiqUserRole.count).to eq(0)
-  end
+  describe "#delete" do
+    let(:role) { FactoryBot.create(:miq_user_role) }
```

Comment:
@yrudman Please just confirm that you deliberately omitted  **_:name => "test role"_** and that the role name is unnecessary.

Reply:
Other than this one inconsistency between the original version and your changes every thing else LGTM 👍

Reply:
`:name` parameter was removed deliberately, we do not need to specify :name if it is not explicitly use in test